include/feed.php

```diff
@@ -177,6 +177,10 @@ function feed_import($xml,$importer,&$contact, &$hub, $simulate = false) {
 	foreach (array_reverse($entrylist) AS $entry) {
 		$item = array_merge($header, $author);
 
+		$alternate = $xpath->query("atom:link[@rel='alternate']", $entry)->item(0)->attributes;
+		if (!is_object($alternate))
```

Comment:
Since this is a bugfix that fixes a nasty bug that really **breaks** things, I would prefer to have it accepted soon without having some "standards ping pong" in this single specific case.